src/chi/components/steps/steps.scss

```diff
@@ -87,5 +92,131 @@
         color: $brand-color;
       }
     }
+
+    &.-horizontalLabels {
+      background-color: set-color(white);
```

Comment:
@davidojedacl remove this background-color as it must be transparent in the base scenario